changelog.txt

```diff
@@ -0,0 +1,6 @@
+fixed - Firestore emulator bug related to https://github.com/firebase/firebase-tools/issues/1073
+fixed - Firestore emulator bug regarding array ordering during writing/reading
+fixed - Firestore emulator handling of query cursors using document names
```

Comment:
For these three: they should be able to read without the "fixed" at the beginning (which is just a tag). So, for example,

"Fixed Firestore emulator's handling of query cursors using document named"

Most of the time, you can just prefix the statement with "Fixed...". Otherwise, the release notes just say "Firebase emulator bug..." - but doesn't say it was _fixed_.